lib/asciidoctor/substitutors.rb

```diff
@@ -728,19 +728,23 @@ def sub_macros(source)
       result = result.gsub(LinkInlineRx) {
         # alias match for Ruby 1.8.7 compat
         m = $~
+        # NOTE if text is non-nil, this the macro syntax
```

Comment:
The changes to substitutors.rb weren't supposed to be part of this PR. I'm going to pull them out into a separate PR because they are addressing something totally different.